js_event_delegation/script.js

```
@@ -1,10 +1,28 @@
 // Answer the questions here:
 
 // - What do you think is going to happen?
-
+// Delegated button will console.log the name of the button.
 // - What happened?
-
+//The button was correctly logged to the console.
 // - Why?
-
+//Because the click delegated the function to the newly created button
```

Pretty much, rather than look at the individual buttons we instead look at the `list` element and let the event bubble until it reaches the target button 👍  Dynamically generated elements are the perfect use case for event delegation with `.on`